website/src/documentation/patterns/side-inputs.md

```diff
@@ -45,4 +45,30 @@ For instance, the following code sample uses a `Map` to create a `DoFn`. The `Ma
 ```java
 {% github_sample /apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java tag:SideInputPatternSlowUpdateGlobalWindowSnip1
 %}
-```
\ No newline at end of file
+```
+
+
+## Slowly updating side input using windowing
+
+You can read side input pcollection periodically into distinct windows.
+Later, when you apply side input to your main input, windows will be matched automatically 1:1.
```

Comment:
Missing "the" -> "when you apply the side input to your main input"

Present tense -> "windows are automatically matched"

It might be less ambiguously to say "each side input window is matched to a main input window" (or something like that), instead of "windows will be matched automatically 1:1." I'm not sure it's clear what it means to be "matched automatically 1:1."